.github/workflows/release.yml

```diff
@@ -0,0 +1,49 @@
+name: release
+
+on:
+  push:
+    tags: [ v*.*.* ]
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      packages: write
+
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      -
+        name: Log into registry ${{ env.REGISTRY }}
+        uses: docker/login-action@f4ef78c080cd8ba55a85445d5b36e214a81df20a
```

Comment:
Idea: if you track the semver tag in a comment, Dependabot should update both!
[example](https://github.com/Shopify/hansel/blob/543acc5e504feea44b8b1268e2bc35d5fa75a8f1/.github/workflows/release.yaml#L17) / [example PR](https://github.com/Shopify/hansel/pull/111).

I really dig it: you get the immutability of pinning a tag, without losing the visibility that the tag provides.

Reply:
oh, neat! will add it! thank you!

Reply:
will be addressed in a separate PR as per: https://github.com/Shopify/kubeaudit/issues/547